Create t3i_face_masks.md (#97)
zuedev/coronavirus-toolkit: docs/tools/t3i_face_masks.md

## T3i: Essential Supplies – Face Masks

<a href="/T3i-Face_Masks_v3.pdf" target="_blank">
    <img class="downloadtools" src="/download-tools.png" />
</a>

### Why is this Tool Helpful to You?

Face masks reduce the spread of virus due to coughing, breathing and talking. Physical barriers like masks are the most effective tool against COVID-19.

### Disclaimer

> The use of information on this site is at reader&#39;s own risk and no party involved in the information production can be held responsible for its use. By using the content found on this website, you further acknowledge that it is not intended to be a substitute for public health agency guidance, professional medical advice, diagnosis, or treatment. Always seek the advice of your physician or other qualified health provider with any questions you may have regarding a medical condition. Never disregard public health agency or professional medical advice or delay in seeking it because of something you have read here.

Reviewer: *the* reader's

> The use of information on this site is at reader&#39;s own risk and no party involved in the information production can be held responsible for its use. By using the content found on this website, you further acknowledge that it is not intended to be a substitute for public health agency guidance, professional medical advice, diagnosis, or treatment. Always seek the advice of your physician or other qualified health provider with any questions you may have regarding a medical condition. Never disregard public health agency or professional medical advice or delay in seeking it because of something you have read here.

Author: ```suggestion
The use of information on this site is at the reader's own risk and no party involved in the information production can be held responsible for its use. By using the content found on this website, you further acknowledge that it is not intended to be a substitute for public health agency guidance, professional medical advice, diagnosis, or treatment. Always seek the advice of your physician or other qualified health provider with any questions you may have regarding a medical condition. Never disregard public health agency or professional medical advice or delay in seeking it because of something you have read here.
```

> The use of information on this site is at reader&#39;s own risk and no party involved in the information production can be held responsible for its use. By using the content found on this website, you further acknowledge that it is not intended to be a substitute for public health agency guidance, professional medical advice, diagnosis, or treatment. Always seek the advice of your physician or other qualified health provider with any questions you may have regarding a medical condition. Never disregard public health agency or professional medical advice or delay in seeking it because of something you have read here.

Author: ```suggestion
The use of information on this site is at the reader's own risk and no party involved in the information production can be held responsible for its use. By using the content found on this website, you further acknowledge that it is not intended to be a substitute for public health agency guidance, professional medical advice, diagnosis, or treatment. Always seek the advice of your physician or other qualified health provider with any questions you may have regarding a medical condition. Never disregard public health agency or professional medical advice or delay in seeking it because of something you have read here.
```

### What You Need

**Preferred:** Surgical mask or homemade HEPA-filter-based mask

**Acceptable:** Other homemade mask (bandana / old scarf / old towel / old t-shirt / 2 elastic bands [+ optional coffee filter])

**Unacceptable:** No mask

### Should I Wear a Mask?

**When leaving the house – Yes.**

**When at home and you are symptomatic or have had recent contact with someone with COVID-19 – Yes.**

**When at home and you have been in lock-down for greater than 14 days and no one at home has symptoms – No.**

There is no harm in wearing a mask covering your face, and it is believed to prevent droplets and a gaseous cloud containing the virus traveling up to 8 meters / 27 feet.

> - COVID-19 is transmitted in droplets or a gasous cloud exhaled by an individual infected with COVID-19

Author: ```suggestion
- COVID-19 is transmitted in droplets or a gaseous cloud exhaled by an individual infected with COVID-19
```

  - These droplets can travel 2 meters or 6 feet away from the individual
  - A gaseous cloud can travel for up to 8 meters / 27 feet
- A mask covering the nose and mouth will prevent or dramatically reduce COVID-19 transmission
- Masks can discourage face-touching, which also protects you against COVID-19

> - Masks are **only effective** when used in combination with frequent hand washing/sanitizing, ssocial distancing, and avoiding face-touching

Author: ```suggestion
- Masks are **only effective** when used in combination with frequent hand washing/sanitizing, social distancing, and avoiding face-touching
```

- You **must** wear a mask when caring for someone suspected of having COVID-19 or if you exhibit symptoms of COVID-19 and must leave the house (e.g. to get tested)
- You **should** wear a mask if you may come into contact with people

> - Children under the age of 2, people who have trouble breathing, unconscious, or are incapacitated **should not** wear a facemask due to risk of suffocation

Reviewer: *are* unconscious

> - Children under the age of 2, people who have trouble breathing, unconscious, or are incapacitated **should not** wear a facemask due to risk of suffocation

Author: ```suggestion
- Children under the age of 2, people who have trouble breathing, are unconscious, or are incapacitated **should not** wear a facemask due to risk of suffocation
```

### Types of Commercial Masks

- Surgical masks are sufficient to prevent COVID-19 transmission
  - The majority of hospital staff use surgical masks when caring for COVID-19 patients
- N95 and FFP3 masks are **not more effective** at preventing respiratory illness transmission than surgical masks ([Greenhalgh and Chan](https://www.cebm.net/covid-19/what-is-the-efficacy-of-standard-face-masks-compared-to-respirator-masks-in-preventing-covid-type-respiratory-illnesses-in-primary-care-staff/))
  - N95 and FFP3 masks only provide increased protection when doctors are working directly with the airway (e.g. intubation, bronchoscopy); N95 or FFP3 masks are often **only** worn by healthcare workers caring for COVID-19 patients during these procedures ([Public Health Ontario](https://www.publichealthontario.ca/-/media/documents/ncov/updated-ipac-measures-covid-19.pdf), [UK PPE Guidelines](https://www.gov.uk/government/publications/wuhan-novel-coronavirus-infection-prevention-and-control/covid-19-personal-protective-equipment-ppe#ppe-guidance-by-healthcare-context) – see Section 8)
  - Members of the public will not be exposed to this increased risk

>   - We do not recommend using N95 masks as it provides you with no additional protection and pulls supply away from essentia healthcare workers

Author: ```suggestion
  - We do not recommend using N95 masks as it provides you with no additional protection and pulls supply away from essential healthcare workers
```

**Source:** [WHO](https://www.who.int/news-room/commentaries/detail/modes-of-transmission-of-virus-causing-covid-19-implications-for-ipc-precaution-recommendations), [CDC](https://www.cdc.gov/coronavirus/2019-ncov/hcp/infection-control-recommendations.html)

> ### Homemade Masks

Reviewer: Home made (two words)

> ### Homemade Masks

Author: ```suggestion
### Home Made Masks
```

> - Homemade cloth masks can be similarly effective to a surgical mask if worn correctly

Author: ```suggestion
- Home Made cloth masks can be similarly effective to a surgical mask if worn correctly
```

</div>

#### Advanced Mask:

- For someone with more time and materials, here is a [video](https://www.youtube.com/watch?v=W6d3twpHwis&amp;feature=youtu.be) and [guide](https://drive.google.com/file/d/1fW_Qrv_1J26Lgd1kawSWqqyKvbYOY2Fu/view) explaining how to construct a safe face mask from a HEPA filter from a vacuum cleaner bag. It is also recommended that these should be donated to local emergency services.

<a href="https://drive.google.com/file/d/1fW_Qrv_1J26Lgd1kawSWqqyKvbYOY2Fu/view" target="_blank">
    <img class="downloadtools" src="/pdf.png" />
</a>

<div class="video-responsive">
  <iframe width="1189" height="669" src="https://www.youtube.com/embed/W6d3twpHwis" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>

**Source:** [CDC](https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/diy-cloth-face-coverings.html), [van der Sande et al., 2008](https://trello.com/c/nh9tuiRU/28-t3i-mask-usage)

> ### Putting-On Masks

Reviewer: Putting On (no hyphen)

> ### Putting-On Masks

Author: ```suggestion
### Putting On Masks
```

Adapted from [WHO Guidelines](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/advice-for-public/when-and-how-to-use-masks)

- Wash your hands with alcohol-based hand rub or soap and water for 20 seconds before putting on a mask
- Cover the mouth and nose with the mask and **ensure there are no gaps** between your face and the mask

### Removing Masks

Adapted from [WHO Guidelines](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/advice-for-public/when-and-how-to-use-masks)

- Remove the mask by removing it from behind, pulling the straps over the ears so that the mask falls forward
- **Do not** touch the front of the mask as it may be contaminated with COVID-19
- Surgical Masks: immediately dispose of them in a closed trash bin

> - Cloth Masks: immediately place in a separate bag after use and se

Reviewer: Sentence is unfinished

> - Cloth Masks: immediately place in a separate bag after use and se

Author: ```suggestion
- Cloth Masks: immediately place in a separate bag after use and set aside for washing in your washing machine
```

- Wash your hands with alcohol-based hand rub or soap and water for 20 seconds

### Reusing Masks

> - Commercial masks (surgical masks) **should not be reused** and should be disposed immediately after use

Reviewer: disposed *of*

> - Commercial masks (surgical masks) **should not be reused** and should be disposed immediately after use

Author: ```suggestion
- Commercial masks (surgical masks) **should not be reused** and should be disposed of immediately after use
```

- Cloth masks **can** be reused but must be cleaned or sterilized before reuse. There are a number of methods available, listed in order of preference:

> - A washing machine using laundry detergent the hot water setting (greater than 70°C/160°F)

Reviewer: This and subsequent bullets should be indented more
*on* the hot water setting

> - A washing machine using laundry detergent the hot water setting (greater than 70°C/160°F)

Author: ```suggestion
- A washing machine using laundry detergent on the hot water setting (greater than 70°C/160°F)
```

> - Soak in 1-part bleach in 50 parts hot water (greater than 70°C/160°F) for greater than 5 minutes, then rinse with water

Reviewer: *and* 50 parts hot water

> - Soak in 1-part bleach in 50 parts hot water (greater than 70°C/160°F) for greater than 5 minutes, then rinse with water

Author: ```suggestion
- Soak in 1-part bleach and 50 parts hot water (greater than 70°C/160°F) for greater than 5 minutes, then rinse with water
```
